src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -38,24 +44,31 @@ const ReportActionItemCreated = (props) => {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
     return (
-        <View
-            accessibilityLabel="Chat welcome message"
-            style={[
-                styles.chatContent,
-                styles.pb8,
-                styles.p5,
-            ]}
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom')}
+            errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom')}
+            errorRowStyles={styles.addWorkspaceRoomErrorRow}
+            onClose={() => Report.navigateToConciergeChatAndDeletePolicyReport(props.report.reportID)}
```

Comment:
any chance report could be `null` here? I'm personally not sure but to be safe let's use `lodashGet`?

Reply:
`report` will always be non-null because of `withOnyx`. It will not render the component until the report is read https://github.com/Expensify/react-native-onyx/blob/bac8d308cb2eaf971174837e30a1f27f6b54c71e/lib/withOnyx.js#L159-L161

Reply:
There might still be something to be careful of here though. 

> It will not render the component until the report is read

While this is true, it does not guarantee that a report is returned. It could read the report from Onyx, find nothing there, then pass `null` as a prop. In fact, you see that `report` is not a required prop.

Reply:
Oh yeah, good point. `report` is a required prop of ReportActionList, which renders the action items and in turn this component, so I think it's safe. ReportActionList also accesses `props.report.reportID` without any checks.